llama_index/logger/__init__.py

```diff
@@ -1,5 +1,14 @@
 """Init params."""
 
+import logging
+from logging import NullHandler
+
 from llama_index.logger.base import LlamaLogger
 
 __all__ = ["LlamaLogger"]
+
+logger = logging.getLogger(__name__)
```

Comment:
Do we want to add any basic formatting? Or should that be left up to the developer

Reply:
Developer seems best? As they would want a single format for their entire application?

Reply:
That makes sense